dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -240,8 +240,8 @@ public String buildInsertCommand(String tableName,
         return sb.toString();
     }
 
-    public DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors) {
-        return new DbStoredProcedureCall(this, name, accessors);
+    public DbStoredProcedure newDbStoredProcedure(String name) {
```

Comment:
Why do you think we would we pass in accessors to the `DbStoredProcedure` constructor? It loads it's own accessors from database metadata.